components/experiments.tsx

```diff
@@ -8,14 +8,14 @@ const ExperimentsSection = () => (
   <StyledExperimentsSection>
     <Container>
       <h2>Experiments</h2>
-      <p className="section-intro max-width">
+      <div className="section-intro max-width">
```

Comment:
we don't need this anymore with your changes in [`eslint-config-upleveled@4.1.1`](https://github.com/upleveled/eslint-config-upleveled/releases/tag/v4.1.1)

Reply:
Updated ESLint and reverted the changes
[0c821df](https://github.com/upleveled/next-starter-peacock/pull/107/commits/0c821dfd7e6682404ef56a45e2cb0ad8cf3fb9bc)